README.md

```diff
@@ -14,7 +14,8 @@ First brought to you in [Buildasaur](https://github.com/czechboy0/Buildasaur), n
 This is an unofficial, community-maintained project and is not associated with Apple. We also maintain the **[unofficial documentation](https://github.com/czechboy0/XcodeServer-API-Docs) of Xcode Server** in case you'd like to integrate using a different programming language! :wink:
 
 # :mortar_board: Getting Started With Xcode Server 
-To find out how to set up Xcode Server on your Mac in minutes (and more), check out my [series of tutorials](http://honzadvorsky.com/pages/xcode_server_tutorials/).
+To find out how to set up Xcode Server on your Mac in minutes (and more), check out my [series of tutorials](http://honzadvorsky.com/pages/xcode_server_tutorials/).  
+If you want to check how easy it is to integrate `XcodeServerSDK` with iOS app, check this little [sample project](https://github.com/cojoj/XCSDemo).
```

Comment:
How about: "To see how easy it is to integrate `XcodeServerSDK` into an iOS app, check out the [XCSDemo](https://github.com/cojoj/XCSDemo) project created by [@cojoj](https://github.com/cojoj)."